contributing.md

```diff
@@ -0,0 +1,72 @@
+# Contributing Guide
+
+## General
+
+1. If you think something belongs in the wrong category, or think there needs to be a new category, feel free to edit things.
+1. Always add your items to the end of a list. The order is first-come-first-serve.
+1. Always add the "Add your item" to the end of each list
+
+`- [Add Your Item](https://github.com/flame-engine/flame-awesome/pulls)`
+
+## Articles
+
+You should put your project in this category if the following conditions are true:
+
+- The Article is about using the Flame Engine.
+- The Article needs to have an English version.
+
+Entries for that category should look like the following:
+
+`- [**My Article**](repository url) - by [Name](url)`
+
+If you have a series of articles, you are allowed to bundle them into a sub header
+
+```
+### Tutorial Topic by Name
+- [**My Article Part 1**](url)
+- [**My Article Part 2**](url)
+```
+
+## Plugins & Libraries
+
+You should put your project in this category if the following conditions are true:
+
+- The project is compatible with the Flame Engine
+- The project contains a `LICENSE` file
+- The documentation `README` contains a description of the project, illustration of the project with a demo or screenshots.
+- The documentation `README` is in English.
+
+Entries for that category should look like the following:
+
+`- [**My Plugin/Library**](repository url) - Short description by [Name](url)`
+
+## Games
+
+1. Open Source is a great learning resource, thus publicly available projects will be listed first.
+1. Make sure to add your Game to the correct Genre.
+1. If you add a new Genre, please order them A-Z.
+
+### Open Source
+
+You should put your project in this category if the following conditions are true:
+
+- Made with the Flame Engine
+- The project contains a `LICENSE` file
+- The documentation `README` contains a description of the project, illustration of the project with a demo or screenshots.
+- The documentation `README` is in English.
+- The project accepts contributions.
```

Comment:
I am not sure that this should be a requirement, I mean the author could not want to accept contributions, but would like to keep open source so people could learn from it?